docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -683,6 +683,53 @@ Where adherence score = 0 for primary or secondary nonadherent; and adherence sc
     - Burn in period will allow the distribution of follow-up appointments to reach equilibrium prior to time start 
 
 
+Creation of "Untreated" SBP Values on Initializaiton
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GBD values for SBP which are used in this sim reflect the US distribution of SBP **including** medication benefits. 
+Therefore, by later applying treatment benefits to certain simulants, we are double counting the population level 
+benefit of treatment. To avoid this, we must add SBP at initialization to create an "untreated" SBP level. 
+
+To do this, we will add a percent increase in SBP to all simulants who are assigned SBP medication at 
+initialization. The section below includes details on who will receive medication. 
+
+In itialization simulants are also assigned to one drug or two drugs. Based on this assignment, a percent increase 
+in SBP level will be given to each. 
+
+.. list-table:: Percent Increase in SBP 
+  :widths: 10 10 
+  :header-rows: 1
+
+  * - Medication Assigned 
+    - Percent Decrease 
```

Comment:
It might be possible to be clearer about the way this medications percent decrease should be used to calculate the untreated value: `untreated * (1 - 0.051) = treated` right?

Reply:
Yes, I agree. I will adjust the language a bit to make the math more clear.